requirements.txt

```diff
@@ -12,3 +12,4 @@ progressbar2
 nltk
 numpy
 requests
+psycopg2==2.6.1
```

Comment:
Okay Kiran, I will remove the version number.

Archana

On Wed, Nov 18, 2015 at 10:58 AM, Kiran Jonnalagadda <
notifications@github.com> wrote:

> In requirements.txt
> <https://github.com/hasgeek/hascore/pull/18#discussion_r45161409>:
>
> > @@ -12,3 +12,4 @@ progressbar2
> >  nltk
> >  numpy
> >  requests
> > +psycopg2==2.6.1
>
> Don't hardcode a version number for dependencies. Our approach is to set a
> minimum version number and revise the code if a later version breaks it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hasgeek/hascore/pull/18/files#r45161409>.
>